chaosaws/__init__.py

```diff
@@ -181,5 +181,7 @@ def load_exported_activities() -> List[DiscoveredActivities]:
     activities.extend(discover_actions("chaosaws.elbv2.actions"))
     activities.extend(discover_probes("chaosaws.elbv2.probes"))
     activities.extend(discover_probes("chaosaws.asg.probes"))
+    activities.extend(discover_actions("chaosaws.awslambda.actions"))
```

Comment:
isn't it better to name the package just 'lambda', not 'awslambda' to keep the consistency?

Reply:
I thought about that but lambda is a reserved word in python

Reply:
It's a good point, but I think `awslambda` is clearer than `lambda_func`, and can see the name collision with simply `lambda`, which would otherwise be preferred.